### PR TITLE
added daily standard deviation calculation

### PR DESCRIPTION
**Pull request by Selkubi**
Tried a minimalist version of a daily standard deviation calculation

### CODE REVIEW

File: inflammation/models.py
Comment (rortizmerino) [inline inflammation/models.py:1]: Does indeed look minimalistic. ::thumbs up::

**Review by rortizmerino — APPROVED** [review]
Pending approval

File: inflammation/models.py [hunk 1]
```diff
@@ -61,3 +61,10 @@ def patient_normalise(data):
     normalised[normalised < 0] = 0
     return normalised
 
+def daily_sd(data):
+    """Calculate the daily sd of a 2D inflammation data array for each day.
```
Comment (marnixvds) [inline inflammation/models.py:65]: Nice docstring, with clear description.

**Review by marnixvds — APPROVED** [review]
Well done, approved.

File: inflammation/models.py
Comment (Selkubi) [reply inflammation/models.py:1]: perks of using functional programming 💯

**Pull request merged**